pkg/dcgmexporter/expcollector.go

```diff
@@ -132,26 +131,28 @@ func (c *expCollector) getMetrics() (MetricsByCounter, error) {
 		uuid = "uuid"
 	}
 	for _, mi := range monitoringInfo {
-		latestValues, err := dcgm.EntityGetLatestValues(mi.Entity.EntityGroupId, mi.Entity.EntityId, c.labelDeviceFields)
-		if err != nil {
-			return nil, err
-		}
-		// Extract Labels
-		for _, val := range latestValues {
-			v := ToString(val)
-			// Filter out counters with no value and ignored fields for this entity
-			if v == SkipDCGMValue {
-				continue
-			}
-
-			counter, err := FindCounterField(c.labelsCounters, val.FieldId)
+		if len(c.labelsCounters) > 0 {
```

Comment:
(optional):
My preference would be:
```
if len(c.labelsCounters) < 1 {
    continue
}
```

Just to avoid extra nesting and improved readability.

Reply:
done!